test/test_retryable_writes.py

```diff
@@ -454,6 +456,48 @@ def test_batch_splitting_retry_fails(self):
         self.assertEqual(coll.find_one(projection={'_id': True}), {'_id': 1})
 
 
+class TestWriteConcernError(IntegrationTest):
+    @client_context.require_version_min(4, 0)
+    @client_context.require_replica_set
+    @client_context.require_no_mmap
+    @client_context.require_failCommand_fail_point
+    def test_RetryableWriteError_error_label(self):
+        listener = OvertCommandListener()
+        client = rs_or_single_client(
+            retryWrites=True, event_listeners=[listener])
+
+        fail_insert = {
+            'configureFailPoint': 'failCommand',
+            'mode': {'times': 2},
+            'data': {
+                'failCommands': ['insert'],
+                'closeConnection': False,
+                'writeConcernError': {
+                    'code': 91,
+                    'errmsg': 'Replication is being shut down'},
+            }}
+
+        if client_context.version < Version(4, 2):
+            # SERVER-39292: specifying closeConnection on MongoDB 4.0+,<4.2
+            # causes the failPoint to fire twice so we remove it.
+            fail_insert['data'].pop('closeConnection')
```

Comment:
Can we just skip adding `closeConnection` for all server versions?

Reply:
Done.